homeassistant/components/tuya/sensor.py

```diff
@@ -839,7 +899,7 @@ def __init__(
 
             # Unknown unit of measurement, device class should not be used.
             if self._uom is None:
-                self._attr_device_class = None
+                self._attr_icon = None
```

Comment:
That is not correct / what I pasted in my comment?

Reply:
🤦 Adjusted!